gremlin-groovy/src/test/java/org/apache/tinkerpop/gremlin/groovy/jsr223/GroovyTranslatorProvider.java

```diff
@@ -222,7 +212,6 @@
 public class GroovyTranslatorProvider extends TinkerGraphProvider {
     @Override
     public GraphTraversalSource traversal(final Graph graph) {
-        final GraphTraversalSource g = graph.traversal();
-        return g.withStrategies(new TranslationStrategy(g, GroovyTranslator.of("g"), true));
+        return graph.traversal();
```

Comment:
given this change, this test and `ParameterizedGroovyTranslatorProvider` really aren't doing anything anymore? where are we testing the new Groovy translator with this rigor? if we are, then may be don't need this anymore?

Reply:
removed. 
I think we have enough coverage for new Groovy translator with `groovy-test` engine.